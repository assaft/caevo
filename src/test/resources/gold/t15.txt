The rains caused the flooding.