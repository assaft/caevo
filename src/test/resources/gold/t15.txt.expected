ei1,rains
ei2,caused
ei3,flooding
l1,ei3,ei1,BEGUN_BY